### ISSUE TITLE
Beginning with Quantum Computing

### ISSUE THREAD

**Issue by darkbits018** [Grimm-s-Alchemy-Chamber/hackers_hive#36]
# Title Of Blog Post : Beginning with Quantum Computing
### Tags : Quantum Computing, Quantum Mechanics.

**Comment by Harikaran1729** [comment]
Hi @darkbits018... Please assign this to me... I would be sending a pull request in the month of October, so that it would be counted in the hactoberfest.. Thank you!!

**Comment by deep-Ak-47** [comment]
yeah sure you can work on it and please refer to contribution guidelines.
